### ISSUE TITLE
File Conflict dialog does not have access keys

### ISSUE THREAD

**Issue by dominoFire** [NuGet/Home#10691]
## Details about Problem

NuGet product used (NuGet.exe | Visual Studio | MSBuild.exe | dotnet.exe): Visual Studio

Product version:  16.9

Worked before? If so, with which NuGet version: 

## Repro steps and/or sample project

1. Create a .NET Framework ASP.NET WebForms project
2. Modify Scripts\bootstrap.js file
3. If bootstrap.js is not installed, install it via PM UI
4. Modify Scripts\bootstrap.js file, add a comment
5. Update bootstrap package via PM UI
6. Following dialog will appear

![image](https://user-images.githubusercontent.com/1192347/112400040-af739b80-8cc4-11eb-8b8c-a0b85c7f6d34.png)

### Expected 
Buttons with Underscore: <ins>Y</ins>es, <ins>N</ins>o, Yes to <ins>A</ins>ll, No to A<ins>l</ins>l
### Actual
No Underscore: Yes, No, Yes to All, No to All

### Notes

This requires localization cycle
